service/start.go

```diff
@@ -119,6 +119,82 @@ func (svc *service) startNostr(ctx context.Context, encryptionKey string) error
 	return nil
 }
 
+func (svc *service) SubscribeToAppRequests(ctx context.Context, appWalletPubKey string) error {
+	relayUrl := svc.cfg.GetRelayUrl()
```

Comment:
Actually, all the relay connection code should not be per app, only setting up subscriptions after relay connection should be per app

Reply:
yep, been refactoring that all morning :)